vcluster/_fragments/sync-from-host-namespaced-custom-resources-example.mdx

```diff
@@ -0,0 +1,189 @@
+import Highlight from "@site/src/components/Highlight/Highlight";
+
+import Flow, { Step } from "@site/src/components/Flow";
+
+
+
+## From host namespaced custom resource example
+
+This guide shows how to sync k8s namespaced custom resources from host cluster.
+Example CRD is used in this guide.
```

Comment:
```suggestion
As an example, CRD syncing is used in this guide.
```

Reply:
I meant here that in this guide we will use Custom Resource called "Example" (that's why it is capitalized).